pkg/tiller/release_testing.go

```diff
@@ -38,13 +38,20 @@ func (s *ReleaseServer) RunReleaseTest(req *services.TestReleaseRequest, stream
 		return err
 	}
 
+	var parallelism uint32
```

Comment:
this can be simplified:

```
parallelism := maxmaxParallelism
if req.MaxParallel == 0 {
	parallelism = maxParallelism
}
```

Reply:
Fixed. I had to do `parallelism := uint32(maxParallelism)` but same effect